[en] Updated the page to be more docker specific (#31421)
kubernetes/website: content/en/docs/tasks/configure-pod-container/pull-image-private-registry.md

<!-- overview -->

This page shows how to create a Pod that uses a
{{< glossary_tooltip text="Secret" term_id="secret" >}} to pull an image 
from a private container image registry or repository. There are many private 

> registries in use. This task uses Docker Hub as an example of the steps needed.

Reviewer: ```suggestion
registries in use. This task uses [Docker Hub](https://www.docker.com/products/docker-hub)
as an example registry.
```

* To do this exercise, you need the `docker` command line tool, and a
  [Docker ID](https://docs.docker.com/docker-id/) for which you know the password.
* If you are using a different private container registry, you need the command 

>   line tool for that registry and any login information for the registry. 

Reviewer: ```sugegstion
* If you are using a different private container registry that uses an equivalent login mechanism,
  you instead need a way to log in to that registry and to write out a configuration file for Docker
  Engine.
```
How about that?

If you use a Docker credentials store, you won't see that `auth` entry but a `credsStore` entry with the name of the store as value.
{{< /note >}}

## Create a Secret based on existing credentials {#registry-secret-existing-credentials}

A Kubernetes cluster uses the Secret of `kubernetes.io/dockerconfigjson` type to authenticate with
a container registry to pull a private image.

> If you are using the Docker Hub regsitry and already ran `docker login`, you can copy that credential into Kubernetes:

Reviewer: (nits)
```suggestion
If you are using the Docker Hub registry and already ran `docker login`, you can copy
that credential into Kubernetes:
```
